x/did/internal/types/events.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+const (
+	EventTypeAddDidDoc     = "create_did_doc"
```

Comment:
```suggestion
	EventTypeAddDidDoc     = "add_did_doc"
```

Reply:
code updated from EventTypeAddDidDoc     = "create_did_doc" to  EventTypeAddDidDoc     = "add_did_doc"